toolkits/global/packages/global-layout-with-sidebar/README.md

```diff
@@ -0,0 +1,67 @@
+# Global layout with sidebar
+
+The `global-layout-with-sidebar` component is a generic layout utility for creating two column layouts where
+
+* One column is a sidebar with a chosen optimal width
+* The other column takes up the rest of the available space
+* The layout automatically wraps into one column based on a minimum percent-based width for the non-sidebar element
+
+The purpose of the `global-layout-with-sidebar` is to enable responsive 1-2 column layouts without the necessity of media query breakpoints. It is based on [Every Layout’s Sidebar](https://every-layout.dev/layouts/sidebar/).
+
+## Branding
+
+The `global-layout-with-sidebar` component can be used in any brand context. You just need to import the component itself:
+
+```scss
+@import '@springernature/global-layout-with-sidebar/scss/10-settings/default';
+@import '@springernature/global-layout-with-sidebar/scss/50-components/layout-with-sidebar';
+```
+
+## Usage
+
+### Default settings
+
+Using the default settings, the `c-with-sidebar` element will be `400px` in the 2-column configuration. Both columns grow to 100% width of their parent `c-with-sidebar` where they wrap into 1 column. By default, this happens when the non-sidebar (“Main content” below) becomes less than `50%` of the total width.
+
+```html
+<div class="c-with-sidebar">
```

Comment:
The classnames in this readme need updating to the new l- prefix

Reply:
@benjclark WHOOPS. Yep, very true. Will quickly do that now.